src/languages/en.js

```diff
@@ -0,0 +1,72 @@
+const en = {
+  translation: {
+    //----MAIN PAGE----
+    navBarHomeTab: 'Home',
+    navBarBackyardTab: 'Backyard',
+    navBarLogoutTab: 'Logout',
+    navBarLoginTab: 'Login',
+    categoryScience: 'Science',
+    categoryAliens: 'Aliens',
+    categoryIlluminati: 'Illuminati',
+    categoryPolitics: 'Politics',
+    categoryCovid: 'Covid',
+    categoryHolliwood: 'Hollywood',
+    dividerTopConspiracies: 'New Top Conspiracies',
+    dividerPremiumArticles: 'Premium Articles',
+    dividerOtherNews: 'Other News',
+    //----BACKYARD DASHBOARD----
+    backyardDashboardHeaderLocationOn: 'Backyard Conspiracies from',
+    backyardDashboardHeaderLocationOff: 'Allow your location!',
+    backyardDashboardTitle: 'Title',
+    backyardDashboardTheme: 'Theme',
+    backyardDashboardWrittenBy: 'Written By',
+    backyardDashboardCreatedOn: 'Created On',
+    backyardDashboardAction: 'Action',
+    backyardDashboardView: 'View',
+    //----LOGIN----
```

Comment:
Is it possible to nest stuff like this? 
```
backyardDashboard: {
  headerLocationOn: 'blbl'
  ...
}
```

Reply:
I don't think so, but we can doublecheck 👯‍♂️

Reply:
Double checked, you can do it. If you do this nesting you need to use following syntax: `t('backyardDashboard.headerLocationOn')`
We don't think that it worth refactoring our code, it is simple but time consuming. But definitely good thing to know in the future.

Reply:
Ye I agree, not worth it at this point probably - but ye, will make it more readable and a little less writing :P